src/reviewer.py

```diff
@@ -0,0 +1,38 @@
+import os
+from github_utils import get_pull_request_diff, post_review_comment
+from llm_utils import analyze_code_changes
+
+class AICodeReviewer:
+    def __init__(self):
+        self.openai_api_key = os.getenv('GEMINI_API_KEY')
+        self.github_token = os.getenv('GIT_TOKEN')
+
+    def review_pull_request(self):
```

Comment:
Consider adding logging to indicate when the review process starts and ends.

```suggestion
|
```print("Starting AI code review...")
try:
...
finally:
print("AI code review complete.")